e2e/cypress/integration/messaging/thread_appears_and_scrollable_in_the_RHS_spec.js

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+//
+// Group: @messaging
+
+import {postListOfMessages} from '../scroll/helpers';
+
+describe('Thread Scrolling Inside RHS ', () => {
+    let channelId;
+    let channelLink;
+    let mainUser;
+    let otherUser;
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiInitSetup().then(({user, team, channel}) => {
```

Comment:
No need to login again as admin since it's already logged in in the global before hook.
```suggestion
    before(() => {
        cy.apiInitSetup().then(({user, team, channel}) => {
```

Reply:
Hi @saturninoabril,
In fact I copied this entire `beforeEach` block from another test.
Now I've reworked all my code considerably (learning Cypress and JS along the way).
Please re-review it

Reply:
@josephbaylon @devinbinnie please re-review too.

Reply:
The good part is the test works almost twice faster )

Reply:
I'm not sure about the [JSDoc section](https://github.com/mattermost/mattermost-webapp/pull/9320/commits/8b93ac1f1827b7bb2d87cf66938095f7a2adaa1a#diff-a46c8a3c058a7f87e9c7d65de3e35d93459c63c1e5497ca5c64b3f9b6efc8367R22-R27) of the new task command.
I basically copied it [from the `postMessageAs` command](https://github.com/mattermost/mattermost-webapp/blob/8b93ac1f1827b7bb2d87cf66938095f7a2adaa1a/e2e/cypress/support/task_commands.js#L7-L9), but the original doesn't include `rootId` & `createAt` params, so I don't know is it really used...

Reply:
Cool, thanks @poflankov!
Those params are used in some cases but I think it's good as is.